src/sentry/static/sentry/app/components/search/sources/apiSource.jsx

```diff
@@ -120,6 +127,42 @@ async function createIntegrationResults(integrationsPromise, orgId) {
   );
 }
 
+async function createSentryAppResults(sentryAppPromise, orgId) {
```

Comment:
Should we start calling these `orgSlug`?

```suggestion
async function createSentryAppResults(sentryAppPromise, orgSlug) {
```

Reply:
@billyvg The problem is we use `orgId` in params as well as in a URL in `handleRequestError`. I can update the variable name in other places but I'm not sure it's worth the effort seeing as how we can't do it in every place.